### ISSUE TITLE
Update Plane Finder client to version 3.7.20 on ARM and 3.7.1 i386

### ISSUE THREAD

**Issue by Romeo-Golf** [jprochazka/adsb-receiver#331]
Updating Plane Finder client to version 3.7.1, which is now consistent across ARM and i386 architectures..

Thanks to @andreapergola for highlighting this in PR #330 :)

**Comment by Romeo-Golf** [comment]
https://planefinder.net/sharing/client_changelog

> 3.7 Latest!
> Improvements to ADS-B decoding
> New GPS view for Plane Finder Radar receivers

**Comment by Romeo-Golf** [comment]
I have pushed this change to my 2.5.1 and 2.6.0 branches, and hope these will be included into @jprochazka's releases, but in the mean time if you would like to assist with testing you can switch to my fork with the following commands:

**Romeo-Golf [2.5.1](https://github.com/Romeo-Golf/adsb-receiver/tree/2.5.1)** 
```
export PROJECTBRANCH=2.5.1
cd ~/adsb-receiver
git remote rm origin
git remote add origin https://github.com/romeo-golf/adsb-receiver.git
git fetch
git checkout -b ${PROJECTBRANCH} origin/${PROJECTBRANCH}
./install.sh
```


**Romeo-Golf [2.6.0](https://github.com/Romeo-Golf/adsb-receiver/tree/2.6.0)**
```
export PROJECTBRANCH=2.6.0
cd ~/adsb-receiver
git remote rm origin
git remote add origin https://github.com/romeo-golf/adsb-receiver.git
git fetch
git checkout -b ${PROJECTBRANCH} origin/${PROJECTBRANCH}
./install.sh -b ${PROJECTBRANCH}
```

Regards.

**Comment by Romeo-Golf** [comment]
FYI testing confirms that 3.7.1 installs cleanly on ARM and while I cant see any visible changes, none were mentioned in the changelog :)

```
Downloading the Plane Finder ADS-B Client package...

Downloading the Plane Finder ADS-B Client v3.7.1 for ARM devices...

--  http://client.planefinder.net/pfclient_3.7.1_armhf.deb
Resolving client.planefinder.net (client.planefinder.net)... 2.22.146.96, 2.22.146.27, 2a02:26f0:4000::216:9260, ...
Connecting to client.planefinder.net (client.planefinder.net)|2.22.146.96|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1167450 (1.1M) [application/vnd.debian.binary-package]
Saving to: ‘/home/pi/adsb-receiver/build/planefinder/pfclient_3.7.1_armhf.deb’

/home/pi/adsb-receiver/build/planef 100%[=====================================================================>]   1.11M  1.87MB/s   in 0.6s   

(1.87 MB/s) - ‘/home/pi/adsb-receiver/build/planefinder/pfclient_3.7.1_armhf.deb’ saved [1167450/1167450]

Installing the Plane Finder ADS-B Client package...

Entering the Plane Finder ADS-B Client build directory...
Installing the Plane Finder ADS-B Client v3.7.1 for ARM devices package...

(Reading database ... 41271 files and directories currently installed.)
Preparing to unpack .../pfclient_3.7.1_armhf.deb ...
Stopping pfclient (via systemctl): pfclient.service.
Unpacking pfclient (3.7.1) over (3.5.469) ...
Setting up pfclient (3.7.1) ...
Processing triggers for systemd (215-17+deb8u6) ...

Checking that the pfclient package was installed properly...
```

**Comment by Calimerorulez** [comment]
Hi Romeo-Golf,

I've switched to your 2.6.0 branch, as I was following your posts lately, and for the lack of updates of the original repository :-)

If I encounter something, I will let you know.

Thank you for your work.

**Comment by Calimerorulez** [comment]
Trying to install AboveTustin on a raspberry pi gives the following error:

E: Package 'ttf-mscorefonts-installer' has no installation candidate

**Comment by Romeo-Golf** [comment]
@Calimerorulez, can you confirm if you are using Raspbian or Ubuntu?...

If its Ubuntu then the following may be relevant:

http://askubuntu.com/questions/659305/package-ttf-mscorefonts-installer-has-no-installation-candidate

**Comment by Calimerorulez** [comment]
I'm on Raspbian, I managed to fix it by 'wget'-ing "http://ftp.de.debian.org/debian/pool/contrib/m/msttcorefonts/ttf-mscorefonts-installer_3.6_all.deb" and manually installing it

**Comment by Romeo-Golf** [comment]
Plane Finder Client for ARM has been updated to 3.7.20:

`$ curl -s https://planefinder.net/sharing/client |grep deb`
`<a href="http://client.planefinder.net/pfclient_3.7.20_armhf.deb" class="download-link generic">`
`<a href="http://client.planefinder.net/pfclient_3.7.1_i386.deb" class="download-link generic">`

**Comment by jprochazka** [comment]
Fixed in commit 569279229b32a3d429196d1be3c39fb0fc1d66fe.
